lib/crawler.js

```diff
@@ -381,12 +385,15 @@ Crawler.prototype.getRequestOptions = function (queueItem) {
         agent: agent,
         headers: {
             "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
-            "Accept-Encoding": "gzip, deflate",
```

Comment:
And it looks like we were always sending this header, so the expected response is probably just to decompress by default.